scripts/install-worker.sh

```diff
@@ -428,24 +429,33 @@ if [[ "$CACHE_CONTAINER_IMAGES" == "true" && "$BINARY_BUCKET_REGION" != "us-iso-
     "${CNI_IMG}:${LATEST_VPC_CNI_VERSION}"
     "${CNI_INIT_IMG}:${LATEST_VPC_CNI_VERSION}"
   )
+  PULLED_IMGS=()
 
-  CACHED_IMGS=(
-    "${PAUSE_CONTAINER}"
-    ${KUBE_PROXY_IMGS[@]}
-    ${CNI_IMGS[@]}
-  )
-
-  for img in "${CACHED_IMGS[@]}"; do
+  for img in "${CACHE_IMGS[@]}"; do
     ## only kube-proxy-minimal is vended for K8s 1.24+
     if [[ "${img}" == *"kube-proxy:"* ]] && [[ "${img}" != *"-minimal-"* ]] && vercmp "${K8S_MINOR_VERSION}" gteq "1.24"; then
       continue
     fi
-    /etc/eks/containerd/pull-image.sh "${img}"
+    ## Since eksbuild.x version may not match the image tag, we need to decrement the eksbuild version until we find the latest image tag within the app semver
+    eksbuild_version="1"
+    if [[ ${img} == *'eksbuild.'* ]]; then
+      eksbuild_version=$(echo "${img}" | grep -o 'eksbuild\.[0-9]\+' | cut -d'.' -f2)
+    fi
+    ## iterate through decrementing the build version each time
+    for build_version in $(seq 1 "${eksbuild_version}" | sort -r -n); do
```

Comment:
I think you can swap the arguments to `seq` to get a descending sequence

Reply:
the mac seq works that way, gnu is a bit trickier, but works better than the reverse sort: 

```
##          FIRST       INC  LAST
seq ${eksbuild_version} -1 1
```